__tests__/app.test.js

```diff
@@ -247,4 +247,22 @@ describe("PATCH:/api/articles/:article_id", () => {
         expect(typeof article.article_img_url).toBe("string");
       });
   });
+
+  test("404: Responds with status 404 and an error message if passed a valid artcile_id which does not exist in the database", () => {
+    return request(app)
+      .patch("/api/articles/3000")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("ID not found");
+      });
+  });
+
+  test("400: Responds with status 400 and an error message if passed an invalid artcile_id", () => {
+    return request(app)
+      .patch("/api/articles/not-an-artcile-id")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid article ID");
+      });
```

Comment:
Really good tests! 🥇 Don't forget we still want our tests to otherwise be valid so we should still send an object to be patched.

Speaking of, what would happen if we sent a bad patch object? 👀